### PR TITLE
Use blob data from big query

### PR DESCRIPTION
**Pull request by torztomasz**
This pull request updates the L2CostsUpdater to fetch blob data from BigQuery instead of using the Ethereum provider. This change improves performance and reduces the number of requests made to the Ethereum provider.

### CODE REVIEW

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/repositories/L2CostsRepository.ts [hunk 1]
```diff
@@ -44,21 +44,10 @@ export class L2CostsRepository extends BaseRepository {
   ): Promise<number> {
     const knex = await this.knex(trx)
     const rows = records.map(toRow)
-    await knex.batchInsert('l2_costs', rows, 10_000)
+    await knex.batchInsert('l2_costs', rows, 1000)
```
Comment (maciekop-l2b) [inline packages/backend/src/modules/tracked-txs/modules/l2-costs/repositories/L2CostsRepository.ts:47]: Is this change intentional or was changed for debugging purposes?

File: packages/backend/src/modules/tracked-txs/modules/liveness/LivenessUpdater.ts [hunk 2]
```diff
@@ -28,12 +28,12 @@ export class LivenessUpdater implements TxUpdaterInterface {
     await this.livenessRepository.addMany(transformedTransactions, knexTx)
   }
 
-  async deleteFrom(
+  async deleteFromById(
     id: TrackedTxId,
     untilTimestamp: UnixTime,
```
Comment (maciekop-l2b) [inline packages/backend/src/modules/tracked-txs/modules/liveness/LivenessUpdater.ts:33]: Let's give it a more suitable name like: fromInclusive currently it's very misguiding

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/api/L2CostsController.ts [hunk 3]
```diff
@@ -384,4 +377,13 @@ export class L2CostsController {
       },
     }
   }
+
+  private weiToEth(wei: bigint): number {
+    // Biggest wei we can get from DB is 2^63-1 which divided by 1e18 is safe to parse to Number
```
Comment (maciekop-l2b) [inline packages/backend/src/modules/tracked-txs/modules/l2-costs/api/L2CostsController.ts:382]: Shouldn't this be `1e9` ?

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsAggregatorIndexer.ts [hunk 4]
```diff
@@ -274,4 +267,13 @@ export class L2CostsAggregatorIndexer extends ManagedChildIndexer {
 
     return multipliers
   }
+
+  private weiToEth(wei: bigint): number {
+    // Biggest wei we can get from DB is 2^63-1 which divided by 1e18 is safe to parse to Number
```
Comment (maciekop-l2b) [inline packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsAggregatorIndexer.ts:272]: Shouldn't this be `1e9` ?

**Review by maciekop-l2b — CHANGES_REQUESTED** [review]
Nice work! Few nitpicks

File: packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsAggregatorIndexer.ts (hunk 4 above)
Comment (torztomasz) [reply packages/backend/src/modules/tracked-txs/modules/l2-costs/indexers/L2CostsAggregatorIndexer.ts:272]: yup

**Review by maciekop-l2b — APPROVED** [review]

**Pull request merged**